crates/transaction-pool/Cargo.toml

```diff
@@ -57,3 +57,4 @@ rand = "0.8"
 default = ["serde"]
 serde = ["dep:serde"]
 test-utils = ["rand", "paste", "serde"]
+optimism = []
```

Comment:
i think we might need to enable any deps with `optimism` feat here as well, not sire